pcfuncs/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mcr.microsoft.com/azure-functions/python:4-python3.8
+
+ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
+    AzureFunctionsJobHost__Logging__Console__IsEnabled=true
+
+
+# TODO: Ask about requirement of v3 for function proxies.
```

Comment:
@mmcfarland the original apis functions in the data catalog used v3 specifically. Was there a specific reason for that, perhaps having to do with Azure Static Web App service?

Reply:
Discussed with Matt in person; there may have been a specific reason for v3 in the development environment, but v4 works for this project and there's no need to bring that forward here.

Reply:
For reference, a previous set of functions (for auth) in the catalog app were using the "function proxies" feature of the v3 runtime, which was removed in v4. That entire branch of function code was removed, but the animation function didn't require that feature regardless.